src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = sourceString =>
+  sourceString
+    .split('\n')
+    .map(rule => rule.trim()
+      .replace(/[\t;]/g, ''))
```

Comment:
You are replacing tab and semicolon characters with a space. But semicolon (;) is used to separate style rules, so removing it this way might cause problems. You should split by semicolon first, then trim and handle each rule separately.